File: minminmax.py
def minMinMax(arr):
    smallest = min(arr)
    largest = max(arr)
    newArr = []
    i = smallest
    while i <=largest:
        i += 1
        newArr.append(i)
    list2 = []
    for el in newArr:
        if el not in arr:
            list2.append(el)
    minAbsent = list2[0]
    return [smallest, minAbsent, largest]

File: minminmax.js
function minMinMax(array) {
    let largest = Math.max(...array)
    let smallest = Math.min(...array)
    let filtered = []
    for (let i = smallest; i <= largest; i++){
      filtered.push(i)
    }
    let filteredArr = filtered.filter(x => !array.includes(x))
   let minAbsent = filteredArr[0]
   return [smallest, minAbsent, largest]
  }